test/TesterInternal/HostedTestClusterBase.cs

```diff
@@ -7,13 +7,16 @@
 
 namespace UnitTests
 {
+    using System.Collections.Generic;
+    using System.Reflection;
```

Comment:
apart from being inside the namespace, I believe these are not needed for the change in this file. I did notice similar a change in several other test files.
EDIT: confirmed by opening the code, these references can go away